CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ include(cmake/ExternalProjects.cmake)
 function(faabric_lib lib_name lib_deps)
     # "Normal" library used for linking internally
     add_library(${lib_name} ${lib_deps})
-    target_include_directories(${lib_name} 
```

Comment:
`clang-format` removes trailing whitespaces, but we have no linting for `CMake` files. Thus the unpleasant and misleading `diff`. I agree though that removing trailing whitespaces is nice. Should we "unofficially" enforce this ourselves @Shillaker ?

Reply:
Yes I think it's great that Jakub is cleaning up all this unnecessary whitespace :joy: . We'll have to do a big clear-up ourselves but this can definitely stay as a start.